spec/features/pdp_spec.rb

```diff
@@ -0,0 +1,184 @@
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_title = find('.product-title').text
+
+      expect(pdp_title).to eq @title
+    end
+
+    it 'has description' do
+      visit router.admin_item_path
+      @description = find('#product_description').value
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_description = find('.well').text
+
+      expect(pdp_description).to eq @description
+    end
+
+    it 'has price' do
+      visit router.admin_item_path
+      @price = find('#product_price').value
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_price = find('.lead.price.selling').text
+
+      expect(pdp_price).to include @price
+    end
+
+    it 'has pictures' do
+      visit router.admin_item_pictures_path
+      @pictures = find('.table.sortable').all('img').collect { |img| img[:src].split('/').last }
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_pictures = find_picture_names(0..1)
+      variants = all('.variant-description').collect(&:text)
+
+      variants.each do |variant|
+        choose variant
+        pdp_pictures += find_picture_names(2..3)
+      end
+
+      expect(pdp_pictures).to eq @pictures
+    end
+
+    it 'has variants' do
+      visit router.admin_item_variants_path
+      @variants = []
+      table = find('.ui-sortable').all('tr')
+      table.each do |variant|
+        within(variant) { @variants << all('td')[1].text }
+      end
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_variants = all('.variant-description').collect(&:text)
+
+      expect(pdp_variants).to eq @variants
+    end
+
+    it 'has properties' do
+      visit router.admin_item_properties_path
+      @properties = all('.form-control').collect(&:value).reject(&:empty?)
+
+      Capybara.reset_session!
+
+      visit router.pdp_path
+
+      pdp_properties = find('#product-properties').all('td').collect(&:text)
+
+      expect(pdp_properties).to eq @properties
+    end
+
+    it 'has titles and breadcrumbs' do
+      visit router.pdp_path
+
+      aggregate_failures do
+        expect(page).to have_css('.breadcrumb')
+        expect(page).to have_css('.product-section-title', text: 'Properties')
+        expect(page).to have_css('.product-section-title', text: 'Variants')
+        expect(page).to have_css('.product-section-title', text: 'Look for similar items')
+      end
+    end
+  end
+
+  context 'Functionality' do
+    before { visit router.pdp_path }
+
+    it 'breadcrumbs redirect to the right page' do
+      breadcrumbs = find('.breadcrumb').all('a').collect(&:text)
+      breadcrumbs.each do |breadcrumb|
+        find('.breadcrumb').click_link breadcrumb
+
+        if breadcrumb == 'Home'
+          expect(page.current_url).to eq "#{Capybara.app_host}/"
+        else
+          expect(page.current_url).to include breadcrumb.downcase
+        end
+
+        go_back
+      end
```

Comment:
I suggest to take out taxons from admin panel and compare them with pdp's breadcrumbs
<img width="397" alt="screenshot 2018-12-19 at 17 16 24" src="https://user-images.githubusercontent.com/9821595/50232658-d5a58500-03b1-11e9-9c1f-d93a2d49ff68.png">